helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/it/HelpDeskBotStories.java

```diff
@@ -41,6 +52,12 @@
   @Autowired
   private ApplicationContext applicationContext;
 
+  @Autowired
+  private HelpDeskSymphonyClient helpDeskSymphonyClient;
```

Comment:
The autowired annotation is not required once you have been created this component outside the spring container.

Reply:
Done.